fluent-plugin-google-cloud.gemspec

```diff
@@ -20,11 +20,11 @@ eos
   gem.require_paths = ['lib']
 
   gem.add_runtime_dependency 'fluentd', '1.11.2'
-  gem.add_runtime_dependency 'googleapis-common-protos', '1.3.9'
+  gem.add_runtime_dependency 'googleapis-common-protos', '= 1.3.10'
```

Comment:
This should be equivalent to `1.3.10`, right? Can we remove `= ` for consistency? Also, what kind of testing have we gone through?

Reply:
Removed.

I haven't done that much extensive testing yet, but this version bump has no actual code changes: https://my.diffend.io/gems/googleapis-common-protos/1.3.9/1.3.10